pycbc/types/timeseries.py

```diff
@@ -251,51 +252,64 @@ def at_time(self, time, nearest_sample=False, interpolate=None):
         interpolate: str, None
             Return the interpolated value of the time series. Choices
             are simple linear or quadratic interpolation.
+        extrapolate: str or float, None
```

Comment:
Why is extrapolate allowed to be a str? Wouldn't giving a str trigger the ValueError on line 280?

Reply:
Yes, at the moment it would, but the idea was that probably some methods would make sense in the future, but won't be implemented yet.